vanilla-framework/vanilla-framework
# Aligned the select styling to match inputs and buttons

## Done
Align the height of selects to match inputs and buttons. Set the vertical spacing to match an input instead of having its own padding.

## QA

- Pull code
- Run `./run serve --watch`
- Open http://0.0.0.0:8101/vanilla-framework/examples/base/forms/selects/
- Open http://0.0.0.0:8101/vanilla-framework/examples/base/forms/input/
- Check they both align 

## Details
Fixes https://github.com/ubuntudesign/vanilla-design/issues/37

## Screenshots
![form inline - vanilla framework - examples](https://user-images.githubusercontent.com/1413534/31693156-06ff17a6-b395-11e7-80f8-56fe25a8828b.png)
